### PR TITLE
[STREAMING][DOCS][EXAMPLES] Minor fixes

### PR DESCRIPTION
**Pull request by jaceklaskowski**

### CODE REVIEW

File: docs/streaming-custom-receivers.md [hunk 1]
```diff
@@ -273,9 +273,9 @@ class CustomActor extends Actor with ActorHelper {
...
 {% highlight scala %}
-// Assuming ssc is the StreamingContext
-val lines = ssc.actorStream[String](Props(new CustomActor()), "CustomReceiver")
+val ssc: StreamingContext = ...
+val lines = ssc.actorStream[String](Props[CustomActor], "CustomReceiver")
 {% endhighlight %}
 
 See [ActorWordCount.scala](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala)
-for an end-to-end example.
+for a complete example.
```
Comment (srowen) [inline docs/streaming-custom-receivers.md]: This was OK as-is; the rest looks OK. Again, can you batch together more changes? this is a lot of little PRs, and this has been discussed several times before

Comment (jaceklaskowski) [inline docs/streaming-custom-receivers.md]: It was discussed, but the changes are a result of my daily code reviews and I don't really know where I end up ahead. I'm now in Streaming so I will...*batch*...more changes next time.

File: docs/streaming-custom-receivers.md [hunk 2]
```diff
@@ -257,9 +257,9 @@ The following table summarizes the characteristics of both types of receivers
 
 ## Implementing and Using a Custom Actor-based Receiver
 
-Custom [Akka Actors](http://doc.akka.io/docs/akka/2.2.4/scala/actors.html) can also be used to
+Custom [Akka Actors](http://doc.akka.io/docs/akka/current/scala/actors.html) can also be used to
```
Comment (rxin) [inline docs/streaming-custom-receivers.md]: linking to current seem like a bad idea given they could have api breaking changes?

**Comment by rxin** [comment]
Other than that one line, rest LGTM.

File: docs/streaming-custom-receivers.md (hunk 2 above)
Comment (jaceklaskowski) [inline docs/streaming-custom-receivers.md]: It's about actors which are the fundamental concept of Akka so it's of less worry (like RDDs in Spark), but have changed it to the Akka version Spark uses, i.e. `2.3.11`.

**Comment by rxin** [comment]
I've merged this.

**Pull request closed without merging**